README.md

```diff
@@ -643,6 +644,20 @@ spec:
 
 **Note**: The `image` and `image_version` are intended for local mirroring scenarios. Please note that using a version of AWX other than the one bundled with the `awx-operator` is **not** supported. For the default values, check the [main.yml](https://github.com/ansible/awx-operator/blob/devel/roles/installer/defaults/main.yml) file.
 
+#### Container security context
+To change/add the security context for a specified container, add your items as key-value pairs under the nbame of the container in question. Possible container names are `redis`, `init`, `ee`, `task`, `web`, `rsyslog`. 
```

Comment:
nbame > name

Reply:
Thanks for spotting it, I have fixed typo in README file.